### PR TITLE
Automatic image creation for HCK-CI

### PR DESCRIPTION
**Pull request by kostyanf14**
Please see details in commit message

### CODE REVIEW

File: auxiliary.ps1 [hunk 1]
```diff
@@ -0,0 +1,104 @@
+$FILTERS = "https://go.microsoft.com/fwlink/?linkid=875139"
+$STUDIOCOMPUTERNAME = "STUDIO"
+$CLIENTCOMPUTERNAME = "CL"
+$KITTYPE = "HLK"
```
Comment (YanVugenfirer) [inline auxiliary.ps1:4]: Should be configurable. I am missing the place where it can be changed.

File: auxiliary.ps1 [hunk 2]
```diff
@@ -0,0 +1,104 @@
+$FILTERS = "https://go.microsoft.com/fwlink/?linkid=875139"
+$STUDIOCOMPUTERNAME = "STUDIO"
+$CLIENTCOMPUTERNAME = "CL"
+$KITTYPE = "HLK"
+$HLKKITVER = 1809
```
Comment (YanVugenfirer) [inline auxiliary.ps1:5]: Should be configurable. I am missing the place where it can be changed.

**Review by YanVugenfirer — COMMENTED** [review]
Another question: what entity process the templates? Is this a subject of future commits?

File: auxiliary.ps1 (hunk 1 above)
Comment (kostyanf14) [reply auxiliary.ps1:4]: I think to use the same method as with `auto.bat`, just edit this file. What do you think about this?

**Comment by kostyanf14** [comment]
@YanVugenfirer 

>what entity process the templates? Is this a subject of future commits?

AutoHCK will use this templates to generate proper file and create ISO from this repo.

File: auxiliary.ps1 (hunk 1 above)
Comment (YanVugenfirer) [reply auxiliary.ps1:4]: Edit by auto_hck?

Comment (kostyanf14) [reply auxiliary.ps1:4]: yes

File: auxiliary.ps1 (hunk 2 above)
Comment (YanVugenfirer) [reply auxiliary.ps1:5]: Let's document it

Comment (kostyanf14) [reply auxiliary.ps1:5]: other variant is create `args.ps1` by analogy with `VirtHCK` to redefine configuration. In this case `auto_hck` will create this file with proper values

Comment (YanVugenfirer) [reply auxiliary.ps1:5]: Looks like a better option. With sample args.ps 1 in repo and documentation.

Comment (kostyanf14) [reply auxiliary.ps1:5]: Updated README is ok as documentation?

Comment (YanVugenfirer) [reply auxiliary.ps1:5]: Perfect

**Review by Basils — APPROVED** [review]

**Pull request merged**